Source/CarthageKit/Git.swift

```diff
@@ -155,26 +155,25 @@ public func checkoutRepositoryToDirectory(
 	_ workingDirectoryURL: URL,
 	revision: String = "HEAD"
 ) -> SignalProducer<(), CarthageError> {
-	return SignalProducer
-		{ () -> Result<[String: String], CarthageError> in
-			var environment = ProcessInfo.processInfo.environment
-			environment["GIT_WORK_TREE"] = workingDirectoryURL.path
-			return .success(environment)
-		}
-		.attempt { _ in
-			Result(attempt: { try FileManager.default.createDirectory(at: workingDirectoryURL, withIntermediateDirectories: true) })
-				.mapError {
-					CarthageError.repositoryCheckoutFailed(
-						workingDirectoryURL: workingDirectoryURL,
-						reason: "Could not create working directory",
-						underlyingError: $0
-					)
-				}
-		}
-		.flatMap(.concat) { environment in
-			return launchGitTask([ "checkout", "--quiet", "--force", revision ], repositoryFileURL: repositoryFileURL, environment: environment)
-		}
-		.then(SignalProducer<(), CarthageError>.empty)
+	return SignalProducer { () -> Result<[String: String], CarthageError> in
+		var environment = ProcessInfo.processInfo.environment
+		environment["GIT_WORK_TREE"] = workingDirectoryURL.path
+		return .success(environment)
+	}
+	.attempt { _ in
```

Comment:
This indentation was actually correct before. We like all the operators (like `.attempt`) to be indented one stop past the line they begin on.